### PR TITLE
Purge the CSS output from Tailwind, etc. so only used styles are published

### PR DESCRIPTION
**Pull request by jaredcwhite**
## Issue

Addresses #1046 & #78

- [x] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description

See comment below for the file size savings.

## Type of change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Prototype/POC (not to merge)
- [x] This change is a refactor/address technical debt
- [ ] This change requires a documentation update
- [ ] This change requires a SQL Script

## How Can This Be Tested/Reviewed?

A/B test of dev vs. this PR on various public/partners pages

## Checklist:

- [ ] My code follows the style guidelines of this project
- [ ] I have performed a self-review of my own code
- [ ] I have reviewed the changes in a desktop view
- [ ] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [ ] I have assigned reviewers
- [ ] I have updated the changelog to include a description of my changes
- [ ] I have run `yarn generate:client` if I made backend changes

### CODE REVIEW

**Comment by jaredcwhite** [comment]
Comparing the Public site's CSS of this PR with dev-bloom.netlify.app using Firefox's Network tab:

It goes from **119KB transferred/2.63MB uncompressed** to **39.69KB transferred/266.11KB uncompressed**. On fast bandwidth that maybe doesn't seem like a huge transfer speed difference, but on slower devices and certainly slowing connections it's a big win!

I eyeballed a number of pages and I _think_ everything's working, but we'll want to do further QA of course.

I'll work on the Partners site next, should be a similar easy config change.

**Comment by jaredcwhite** [comment]
Oh good, looks like the output's getting minified as well now (see comparison of the partners' dev site and this PR)

Screen Shot 2021-09-13 at 7 14 12 PM

File: sites/partners/tailwind.config.js [hunk 1]
```diff
@@ -8,4 +8,15 @@ const bloomTheme = cloneDeep(require("@bloom-housing/ui-components/tailwind.conf
 // For example:
 // bloomTheme.theme.colors.white = "#f0f0e9"
 
-module.exports = bloomTheme
+module.exports = {
+  ...bloomTheme,
+  purge: {
+    enabled: true, // in future we can remove the explicit enabling and purge only in production
```
Comment (emilyjablonski) [inline sites/partners/tailwind.config.js:14]: why not do the purging only in prod now?

Comment (jaredcwhite) [reply sites/partners/tailwind.config.js:14]: We could, just not sure how confident we are that this is working perfectly. If it purges in development we can see immediately if something is off when working on the codebase. But if folks think we should change this, fine with me.￼

**Pull request merged**